platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/aibot/ChatPresenter.kt

```diff
@@ -36,7 +36,7 @@ class ChatPresenter : Presenter<ChatScreen.State> {
           isLoading = true
           val response = Message(callApi(event.message), isMe = false)
           messages = messages + response
-          isLoading = false
+          //          isLoading = false commented out to test loading animation
```

Comment:
Let's not leave this in when merging

Reply:
Thanks, fixed here: 5f8baf949662c19f7c1ae23ceba5b5559e9de40f